frontend/src/features/public-form/utils/inputTransformation.ts

```diff
@@ -187,11 +187,18 @@ const transformToSectionOutput = (
 
 const transformToChildOutput = (
   schema: ChildrenCompoundFieldSchema,
-  input: ChildrenCompoundFieldValues,
+  input?: ChildrenCompoundFieldValues,
 ): ChildBirthRecordsResponse => {
+  const noOfChildrenSubFields = schema.childrenSubFields?.length ?? 1
+  let answerArray: string[][]
+  if (input !== undefined && input.child) {
```

Comment:
Nit: Looking at being terser, would `if (input?.child) {` be sufficient?

Reply:
[9089e80](https://github.com/opengovsg/FormSG/pull/6673/commits/9089e8026f129e21341f060d664758b08223276d)